test/test_torch.py

```diff
@@ -4646,6 +4646,12 @@ def test_dlpack_conversion(self):
         z = from_dlpack(to_dlpack(x))
         self.assertEqual(z, x)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "No CUDA")
+    def test_dlpack_cuda(self):
+        x = torch.randn(1, 2, 3, 4).cuda()
+        z = from_dlpack(to_dlpack(x))
+        self.assertEqual(z, x)
```

Comment:
This is unlikely to actually test for the bug, because the act of allocating the cuda tensor will initialize CUDA... so you won't hit it. But it's annoying to actually trigger the bug (you need an out-of-process test) so this seems fine.

Reply:
yeah, made sure it doesn't collapse, but hard to test.